rdkit/Chem/PeriodicTable.py

```diff
@@ -128,11 +127,11 @@
 for line in periodicData.split('\n'):
   splitLine = line.split()
   if len(splitLine) > 1:
-    nameTable[splitLine[1]] = (int(splitLine[0]),float(splitLine[6]),int(splitLine[7]),\
-                               int(splitLine[5]),float(splitLine[2]),float(splitLine[3]),
+    nameTable[splitLine[1]] = (int(splitLine[0]), float(splitLine[6]), int(splitLine[7]),
+                               int(splitLine[5]), float(splitLine[2]), float(splitLine[3]),
                                float(splitLine[4]))
-    numTable[int(splitLine[0])] = (splitLine[1],float(splitLine[6]),int(splitLine[7]),\
-                                   int(splitLine[5]),float(splitLine[2]),float(splitLine[3]),
+    numTable[int(splitLine[0])] = (splitLine[1], float(splitLine[6]), int(splitLine[7]),
```

Comment:
It's great that this is possible; it allows us to solve the problem when someone makes a mistake and combines formatting changes with substantive changes. I would still prefer to not have that happen